.openpublishing.redirection.json

```diff
@@ -2408,6 +2408,26 @@
             "source_path": "docs/framework/windows-workflow-foundation/samples/property-grid-extensibliity.md",
             "redirect_url": "/dotnet/framework/windows-workflow-foundation/samples/property-grid-extensibility",
             "redirect_document_id": true
+        },
+        {
+            "source_path": "docs/framework/wpf/controls/passwordbox-syles-and-templates.md",
```

Comment:
we try to add entries in alphabetical order

Reply:
Rebased and put them by the other "syles" redirect